editor/plugins/org.fusesource.ide.project/plugin.xml

```diff
@@ -87,28 +87,6 @@
 	</extension>
 
  <extension
-       point="org.eclipse.ui.commands">

```

Comment:
I already deleted the other ones and thought thats it but it seems many entries here have been duplicated which then caused the issue.